<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_4" >
        <attribute name="title" >
         <string>Main</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Main settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8" >
            <item>
             <widget class="QCheckBox" name="mainInitialized" >
              <property name="text" >
               <string>Restore default settings on next start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="mainFocusFollowsMouse" >
              <property name="text" >
               <string>Use 'focus follows mouse' between widgets</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="mainShowStatusbar" >
              <property name="text" >
               <string>Show statusbar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Initial size (unless set by window manager):</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" >
              <item>
               <widget class="QLabel" name="label_5" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mainInitialX" >
                <property name="maximum" >
                 <number>9000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="text" >
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mainInitialY" >
                <property name="maximum" >
                 <number>9000</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="mainSaveSizeOnExit" >
              <property name="text" >
               <string>Save size on exit</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout" >
              <item row="0" column="1" >
               <widget class="QLineEdit" name="mainExternalEditor" />
              </item>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Picture editor:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5" >
        <attribute name="title" >
         <string>Directory browser</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Directory view</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <item>
             <widget class="QCheckBox" name="dirShowOnlyDirs" >
              <property name="text" >
               <string>Show only directories</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="dirShowSizeCol" >
              <property name="text" >
               <string>Show 'size' column</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="dirShowTypeCol" >
              <property name="text" >
               <string>Show 'type' column</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="dirShowLastModifiedCol" >
              <property name="text" >
               <string>Show 'last modified' column</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Thumbnail viewer</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Thumbnail view</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6" >
            <item>
             <widget class="QCheckBox" name="tnShowOnlyFiles" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Show only files</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="tnFilterFiles" >
              <property name="text" >
               <string>Filter files</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="tnCaseInsensitiveMatching" >
              <property name="text" >
               <string>Use case-insensitive matching</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2" >
              <item row="0" column="0" >
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>Selection mask:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QLineEdit" name="tnFileMask" />
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3" >
        <attribute name="title" >
         <string>Viewer</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Viewer settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9" >
            <item>
             <widget class="QCheckBox" name="viewerFitToWindow" >
              <property name="text" >
               <string>Fit image to window per default</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="viewerShrinkOnly" >
              <property name="text" >
               <string>Shrink images only when fitting to window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="viewerSmoothTransformation" >
              <property name="text" >
               <string>Use smooth transformation (slower)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="viewerResetFtwOnChange" >
              <property name="text" >
               <string>Reset 'fit to window'-flag on change to next picture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="viewerHideInfoArea" >
              <property name="text" >
               <string>Hide information area</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>On Startup: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="viewerLoadAction" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text" >
                  <string>Load default image</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Load last image</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Don't load anything</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QDialogButtonBox" name="buttonBox" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
